src/Application/StripeChatterFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace MatchBot\Application;
+
+use Symfony\Component\Notifier\Bridge\Slack\SlackTransport;
+use Symfony\Component\Notifier\Chatter;
+use Symfony\Component\Notifier\ChatterInterface;
+
+class StripeChatterFactory
+{
+    private readonly string $apiToken;
+
+    public function __construct(#[\SensitiveParameter] string $apiToken)
```

Comment:
SensitiveParameter doesn't work until PHP 8.2, but we may as well write it in advance.